packages/next/experimental-script.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./dist/client/experimental-script')
```

Comment:
Let's make this export undefined unless you opt-into an experimental flag, just to make sure no one relies on this.

Reply:
Added this logic in [`webpack-config.js`](packages/next/build/webpack-config.ts)